common/scala/src/main/resources/application.conf

```diff
@@ -128,6 +128,10 @@ whisk {
         logs-enabled       = ${?METRICS_LOG}
     }
 
+    zookeeper {
+        prefix = ""
```

Comment:
How about adding a comment to describe it should start without `/` and end with `/`?